nautobot/extras/test_jobs/fail.py

```diff
@@ -7,15 +7,15 @@
 logger = get_task_logger(__name__)
 
 
-class TestFail(Job):
+class TestFailJob(Job):
```

Comment:
I renamed `TestFail` and `TestPass` to `TestFailJob` and `TestPassJob` so that the `generated-members` config added in pyproject.toml for `.*Job.request` would stop `no-member` from complaining about their multiple references to `self.request`.